flink-scala/src/main/scala/org/apache/flink/api/scala/ExecutionEnvironment.scala

```diff
@@ -40,6 +40,7 @@ import org.apache.hadoop.mapred.{FileInputFormat => MapredFileInputFormat, Input
 import org.apache.hadoop.mapreduce.lib.input.{FileInputFormat => MapreduceFileInputFormat}
 import org.apache.hadoop.mapreduce.{InputFormat => MapreduceInputFormat, Job}
 
+import scala.annotation.implicitNotFound
```

Comment:
Ah true. Why is it actually that our Scala Checkstyle does not detect unused imports? Not supported or not configured?